components/prism-false.js

```diff
@@ -0,0 +1,32 @@
+(function (Prism) {
+	/**
+	 * Implements.
+	 *
+	 * @see {@link https://github.com/PrismJS/prism/issues/2801#issue-829717504}
+	 */
+	Prism.languages['false'] = {
+		'comment': {
+			pattern: /\{[^}]*\}/
+		},
+		'string': {
+			pattern: /"[^"]*"/,
+			greedy: true
+		},
+		'character-code': {
+			pattern: /'[\S\s]/,
+			alias: 'number'
+		},
+		'assembler-code': {
+			pattern: /\d+`/,
+			alias: 'important'
+		},
+		'number': /\d+/,
+		'operator': /[-!#$%&'*+,./:;=>?@\\^_`|~\xdf\xf8]/,
```

Comment:
It's more readable to use the characters directly. Our text files are UTF-8 encoded, so special characters aren't a problem.

```suggestion
		'operator': /[-!#$%&'*+,./:;=>?@\\^_`|~ßø]/,
```

Reply:
Got it!